forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt

```diff
@@ -0,0 +1,30 @@
+Name:Urza, Lord Protector
+ManaCost:1 W U
+Types:Legendary Creature Human Artificer
+PT:2/4
+S:Mode$ ReduceCost | ValidCard$ Artifact,Instant,Sorcery | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Artifact, instant and sorcery spells you cast cost {1} less to cast.
+A:AB$ Meld | Cost$ 7 | ConditionPresent$ Card.Self+YouCtrl+YouOwn | ConditionCheckSVar$ X | Name$ Urza, Planeswalker | Primary$ Urza, Lord Protector | Secondary$ The Mightstone and Weakstone | SecondaryType$ Artifact | SorcerySpeed$ True | SpellDescription$ If you both own and control CARDNAME and an artifact named The Mightstone and Weakstone, exile them, then meld them into Urza, Planeswalker. Activate only as a sorcery.
+SVar:X:Count$Valid Artifact.YouCtrl+YouOwn+namedThe Mightstone and Weakstone
+DeckHints:Name$The Mightstone and Weakstone & Type$Artifact|Instant|Sorcery
+MeldPair:The Mightstone and Weakstone
+AlternateMode:Meld
+Oracle:Artifact, instant and sorcery spells you cast cost {1} less to cast.\n{7}: If you both own and control Urza, Lord Protector and an artifact named The Mightstone and Weakstone, exile them, then meld them into Urza, Planeswalker. Activate only as a sorcery.
+
+ALTERNATE
+
+Name:Urza, Planeswalker
+ManaCost:no cost
+Colors:white,blue
+Types:Legendary Planeswalker Urza
+Loyalty:7
+S:Mode$ Continuous | EffectZone$ Battlefield | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME's loyalty abilities can be activated twice each turn rather than only once | CharacteristicDefining$ True | Description$ You may activate the loyalty abilities of CARDNAME twice each turn rather than only once.
```

Comment:
Not CharacteristicDefining

Reply:
Forgot to remove during testing, thanks!